### PR TITLE
doc: add minutes for Sep 27 2017

### PR DESCRIPTION
**Pull request by mhdawson**

### CODE REVIEW

File: meetings/2017-09-27.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+
+## Minutes
+
+### Announcements
+No announcements this week
+
+### Workgroup Update
+
+#### Release Work Group
+* I must have miscommunicated (or just not provided dial-in info) so we
```
Comment (sam-github) [inline meetings/2017-09-27.md]: once this lands it won't be clear who "I" is

**Comment by rvagg** [comment]
Nice first-pass at the values @mylesborins, :thumbsup:

File: meetings/2017-09-27.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+    * Based on discussion in the TSC meeting consensus is that we can
+      move forward. We can land once comments are addressed unless
+      there are objections in next 48 hours.
+
+### nodejs/TSC
+
+### nodejs/node 
+* [#15520](https://github.com/nodejs/node/pull/15520)
+  * May be issue/bug on centos 5
+  * Rich, some discussion of dropping centos 5 so maybe we
```
Comment (Trott) [inline meetings/2017-09-27.md]: This looks cut-off. Maybe delete `so maybe we`?

Comment (mhdawson) [reply meetings/2017-09-27.md]: Done

**Comment by mhdawson** [comment]
Landed as c5e6fb204782dcc76fe04d9fc8e498682dcdaa7d

**Pull request closed without merging**